domains/amardeep.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "Amardeep Lakshkar Portfolio",
+    "repo": "https://github.com/amardeeplakshkar/portfolio",
+    "owner": {
+      "username": "amardeeplakshkar",
+      "email": "amardeep.devs@gmail.com",
+      "discord": ""
```

no need field if its empty
```suggestion
      "email": "amardeep.devs@gmail.com"
```